src/styles/calendar.scss

```diff
@@ -0,0 +1,47 @@
+.calendar {
+  margin: auto;
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+  gap: $gap-size;
+  padding: 50px 10px;
+
+  &__day {
+    width: $cell-size;
+    height: $cell-size;
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    border: 1px solid #000;
+    border-radius: 10%;
+    transition: $time-transform;
```

Comment:
Please specify which properties you are transitioning, like transform, color, etc.
![image](https://user-images.githubusercontent.com/118751034/209431691-cdbde142-0d45-45c4-a525-a5beaa615626.png)

Reply:
$transform-bgcolor-color, so ok?